packages/extension-commerce-bm-seo/README.md

```diff
@@ -0,0 +1,91 @@
+# SEO Extension
+
+A [PWA Kit](https://github.com/SalesforceCommerceCloud/pwa-kit) extension that adds SEO functionality to your application through the [Shopper SEO URL Mapping API](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-seo?meta=getUrlMapping). This extension provides features like:
+
+- Integrate routes managed in Business Manager with the PWA Kit
+- Integration with SCAPI Shopper SEO URL Mapping API
+- Support for multiple resource types (products, categories)
+
+## Installation
+
+```sh
+npm install @salesforce/extension-commerce-bm-seo
+
+# Also: 
+# - install the peer dependencies listed in the package.json
+# - see the Peer Dependancies section below for any other steps (e.g. make sure your app uses CommerceApiProvider component)
+```
```

Comment:
The peer dependencies automatically get installed. The fact that we have to do that when testing in our mono-repo is a side effect of lerna not being able to resolve peer dependencies correctly. So we don't really have to include this.